docs/templates/PACKAGE_MAIN_DOCUMENTATION.MD

```diff
@@ -0,0 +1,105 @@
+**📦 Resources**
+
+<!-- RESOURCES:
+* required
+
+Listing of resources related to the topic including running versions of used code snippets or other applications. 
+Resource links are prefixed with the "🔗" icon
+-->
+
+Example:
+> [🔗 GitHub](https://github.com/BioPhoton/rx-angular-cdk-coercing).
+
+
+# ⛽ Motivation
+
+<!-- MOTIVATION:
+* required
+
+Motivation is here to point out the value for the consumer. If possible we can make clear in which situations the impact will get visible most.
+
+Importent to cover in this section is:
+- Theoretical background to understand the problem
+- The problem and pain points in detail
+- Sketch a optimal solution for the problem
+-->
+
+Example:
+> 
+
+# Available Approaches
+<!-- AVAILABLE_APPROACHES:
+* required
+
+This section should sum up current available approaches and point out the mors importat differences.
+The listing can be done in a bullet point list with optional links.
+-->
+
+Example:
+> - Disabling zone.js completely (compiler options)
+> - Disabling specific API's (zone configuration)
+> - Overriding Angular dependencies (custom event manager)
+> - Using `unpatch`
```

Comment:
If we stick to that section I would at least swap it with the RxAngular solution. This should be the first people get to see.

Reply:
my point of view: I don't want users to scroll a lot until they finally see the RxAngular solution. It is most important for the doc, so it should be at the most prominent position